Extensions/xkit_patches.js

```diff
@@ -227,13 +227,51 @@ XKit.tools.make_gist = function(text, name) {
 			public: false,
 			files: files,
 		}),
-		// '{"description": "a gist for a user with token api call via ajax","public": false,"files": {"file1.txt": {"content": "testing 1 2 3"}}}'
 	}).then(function(resp){
 		return resp.html_url;
 	});
 };
 
 /**
+ * Creates a link to a github issue with error text and template
+ * @param {String} title - the title of the github issue--should be unique and useful
+ * @param {Object?} data - Key-value pairs to list at the top of the issue.
+ * @param {Error?} error - An exception to serialize, if availible
+ * @return {String} The url to link the user to
+ */
```

Comment:
This might need to include changes to the docs too? not sure what our status is on doc generation.